physical_properties/pKa/submission_templates/typeIII_macroscopic_pKas.csv

```diff
@@ -0,0 +1,44 @@
+# Type III Submissiion
+#
+# PREDICTION SECTION
+#
+# Add new lines for each predicted molecule.
+# Leave csv sections you don't have estimates for empty.
+# Report pKas with 2 decimal places.
+# 
+# The data in each prediction line should be structured as follows:
+# Molecule ID, macroscopic pKa, macroscopic pKa SEM
```

Comment:
So what happens if there are multiple macroscopic pKas for the same molecule?

Reply:
Seems like this needs to be answered.

Reply:
Seems like this needs answering.

Reply:
They must add a new line. Added explanation for this.

Reply:
Done. I added:
"If you predict multiple macroscopic pKas for a molecule, add new lines for each pKa with the same molecular ID as identifier(first column)."